code/modules/uplink/uplink_item.dm

```diff
@@ -222,6 +222,16 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	item = /obj/item/weapon/gun/ballistic/automatic/c20r/unrestricted
 	include_modes = list(/datum/game_mode/gang)
 
+/datum/uplink_item/dangerous/minigun
+	name = "M134 Minigun"
+	desc = "She weighs one hundred fifty kilograms and fires two hundred dollar, custom-tooled cartridges at ten rounds per second."
+	item = /obj/item/weapon/minigunpack/ballistic
+	surplus = 0
+	cant_discount = TRUE	//No.
+	player_minimum = 40		//No lowpop minigunning.
+	include_modes = list(/datum/game_mode/nuclear)
+	cost = 55
```

Comment:
@KorPhaeron What this lacks in direct burst damage it more than makes up in being near infinite for ammo, and the cost is almost as high (or higher) than the SAW because with the SAW you quickly run out of bullets.